.eslintignore

```diff
@@ -0,0 +1,2 @@
+examples
```

Comment:
Why are we ignoring the examples dir? This is code that we're constantly going to be pointing people at so we'd surely want it to be formatted correctly?

Reply:
I'd agree, but a lot of the frameworks come with their own default linting rules (some tslint, some eslint) with particular plugins such as react hooks. e.g. see under angular, next etc.

I'd have to look into whether its possible to define a ruleset that could cover all of them at once.

Reply:
Sure, not a big deal I guess. Can tackle it later if needed.